Processes/GitHub-Workflow-to-Publish-Documentation.md

```diff
@@ -0,0 +1,145 @@
+# How To Publish EOS Documentation in GitHub
+
+This document provides instructions to use GitHub to publish documentation to the [EOS Documentation portal](https://docs.eosnetwork.com).
+
+## Definitions
```

Comment:
We should not be teaching about github in our docs. 

We can specify that "main" is our master branch, but we do not need to go into such detail. The internet is plagued with sources to get people up to date about using github.

Reply:
Really, this entire doc is just about how to use github. I'd prefer to remove it. 
This kind of stuff (interface) is much better explained visually, if we want to take that route in the future.

Reply:
Deleted doc

Done